src/__snapshots__/index.test.js.snap

```diff
@@ -28,6 +28,14 @@ exports[`ifWindows as darwin 1`] = `"echo alternate"`;
 
 exports[`ifWindows as win32 1`] = `"echo main"`;
 
+exports[`includePackage as darwin 1`] = `"{\\"root\\":\\"cd packages/client && npm start root\\",\\"nested\\":{\\"child1\\":\\"cd packages/client && npm start nested.child1\\",\\"child2\\":{\\"script\\":\\"cd packages/client && npm start nested.child2 && cd /\\"C://dev//me//experiments//nps-utils/\\"\\",\\"description\\":\\"this is a description\\"},\\"child3\\":{\\"deep\\":{\\"script\\":\\"cd packages/client && npm start nested.child3.deep && cd /\\"C://dev//me//experiments//nps-utils/\\"\\",\\"description\\":\\"this is a description\\"}}}}"`;
```

Comment:
Looks like you're on a windows machine, so that's why the snapshots are failing. We need to serialize it differently to make sure things are working properly regardless of the platform. That's what `relativeizePath` is supposed to do. Could you figure out why it's not doing that?

Actually... Now that I think of it, I think this is a bit of a different kind of feature so I don't think I want to take a snapshot of it like we do with the others. Why don't you add a `test` block at the bottom of the file rather than include it in the `snapshotTests` object...

Reply:
Ye I thought so. It doesn't really seem to fit the snapshots quite so much. Ill write some tests for it instead.

Reply:
Im struggling with testing this as im not sure how to mock "require" correctly.

```import {includePackage} from './index'

test('it runs the default on a package', () => {
  const packageName = 'foo'
  jest.mock(`./packages/${packageName}/package-scripts.js`, () => ({
    scripts: {
      default: 'echo hello',
    },
  }))
  const outp = includePackage(packageName)
  expect(outp).toBe('{"default": "cd packages/foo && npm start default"')
})
```

Any tips?

Reply:
Hi @mikecann, don't put a `jest.mock` call inside a `test`. It doesn't work like that. Put it at the top of the file. Also, I'm not entirely sure how things work with mocking files that don't actually exist... You may have to make a `__fixtures__` directory and put real files in there.

I'm sorry. I have very limited time to give to maintaining this package :-/ Good luck.